### DISCUSSION TITLE
posts/harry-potter-and-utf8

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#17] [General]
# posts/harry-potter-and-utf8

在B站看哈利波特电影，弹幕里竟然能发死亡圣器的符号。这篇文章分析了背后的原理，并且教你如何利用编码规则拼出你想要的标志。

https://www.whexy.com/posts/harry-potter-and-utf8

**Comment by whexy** [comment]
刚看完神奇动物在哪里3，一句话评价：不要去看。
